lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -497,7 +493,13 @@ export class StandardController implements IController {
                     EventType.HANDLE_REDIRECT_END,
                     InteractionType.Redirect
                 );
-                rootMeasurement.end({ success: false });
+
+                rootMeasurement.end(
+                    {
+                        success: false,
+                    },
+                    new AuthError("no_server_response")
```

Comment:
I'm a bit confused by this. No response is not an error. Also in the if above we're setting success to true, but it looks like this will overwrite and success will always be false no matter what?